src/ILCompiler.Compiler/src/IL/ILImporter.Scanner.cs

```diff
@@ -275,7 +277,7 @@ private void ImportCall(ILOpcode opcode, int token)
 
             // If we're scanning the fallback body because scanning the real body failed, don't trigger cctor.
             // Accessing the cctor could have been a reason why we failed.
-            if (!_isFallbackBodyCompilation)
+            if (!_isFallbackBodyCompilation && !method.Signature.IsStatic)
```

Comment:
I do not think that RyuJIT ever adds cctor triggers at the callsite. Where do you see it happening?